src/docfx/lib/schema/JsonSchema.cs

```diff
@@ -34,6 +34,14 @@ internal class JsonSchema
         /// </summary>
         public JValue[] Enum { get; set; }
 
+        /// <summary>
+        /// An object can have extra keys not defined in properties.
+        /// This can be:
+        ///     - boolean: allow/disallow additional properties
+        ///     - object: the schema for the additional properties
+        /// </summary>
+        public JToken AdditionalProperties { get; set; }
```

Comment:
We can only support `bool` for now

```suggestion
        public bool AdditionalProperties { get; set; }
```

or write an `ObjectOrValueConverter` similar to `OneOrManyConverter`

```suggestion
        [JsonConverter(typeof(ObjectOrValueConverter))]
        public (JsonSchema obj, bool value) AdditionalProperties { get; set; }
```